### ISSUE TITLE
[A11Y] User is unable to navigate between the datagrid single action menu items using the up/down arrow keys

### ISSUE THREAD

**Issue by KingMario** [vmware/clarity#6566]
## Describe the bug

User is unable to navigate between the datagrid single action menu items using the up/down arrow keys.

## How to reproduce

1.  Go to https://stackblitz.com/edit/clarity-light-theme-v5-datagrid-simple?file=src%2Fapp%2Fapp.component.html
2.  Click on the vertical '...' of each row of the datagrid
3.  See if user can navigate between the actions of the actions menu

## Expected behavior

According to https://www.w3.org/WAI/GL/wiki/Using_ARIA_menus#Keyboard_Actions_if_focus_is_on_a_menu_item, user should be able to navigate between the actions using the up/donw arrow keys.

## Versions

**Clarity project:**

- [ ] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [ ] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Comment by gnomeontherun** [comment]
@KingMario are these issues tracked by our internal a11y team? If so that is where we review and triage issues so we don't have duplicate issues. If they are not tracked by our a11y team, they should be. Adding @scroniser for visibility.

**Comment by Jinnie** [comment]
This is internally tracked
